NServiceBus.SagaScheduler/ScheduleSaga.cs

```diff
@@ -39,7 +39,7 @@ public Task Timeout(SchedulerTimeOut state, IMessageHandlerContext context)
         {
             //Got a timeout message. Recheck legacy system.
             Console.ForegroundColor = ConsoleColor.White;
-            Console.WriteLine("Running legacy system job");
+            Console.WriteLine($"[{DateTime.Now:T}] Running legacy system job. Timeout requested at {state.RequestedAt:T}");
```

Comment:
I put this in here because I was confused at first about why both endpoints were handling messages. But then it was nice to see when the timeout was started and when it was actually handled, and by which console window.